content/docs/rfcs/20/README.md

```diff
@@ -13,7 +13,7 @@ contributors:
 - Public Key Broadcast: `/eth-dm/1/public-key/json`,
 - Direct Message: `/eth-dm/1/direct-message/json`.
 
-This specification explains the Ethereum Direct Message protocol
+This specification explains the Toy Ethereum Direct Message protocol
```

Comment:
Maybe add a disclaimer about crypto not being audited etc and just serves as an example, not for production usage